### PR TITLE
fix(scully): lint issues and typos

### PR DESCRIPTION
**Pull request by Splaktar**
- remove some unused imports
- shorten some import paths
- fix variables with the same name in the same scope

### CODE REVIEW

File: scully/renderPlugins/contentRenderPlugin.ts [hunk 1]
```diff
@@ -62,6 +53,8 @@ async function handleFile(extension: string, fileContent: string) {
 }
 
 function getScript() {
-  return `<script>try {window['scullyContent'] = document.body.innerHTML.split('<!--scullyContent-begin-->')[1].split('<!--scullyContent-end-->')[0];} catch(e) {console.error('scully could not pare content',e);}</script>
-`;
+  return `<script>
```
Comment (SanderElias) [inline scully/renderPlugins/contentRenderPlugin.ts]: This was a single line on purpose.  Its a bit more readable like this, I agree, but its not meant to be read ;)
I wrote it to be as small as possible.

File: scully/renderPlugins/contentRenderPlugin.ts [hunk 2]
```diff
@@ -62,6 +53,6 @@ async function handleFile(extension: string, fileContent: string) {
 }
 
 function getScript() {
-  return `<script>try {window['scullyContent'] = document.body.innerHTML.split('<!--scullyContent-begin-->')[1].split('<!--scullyContent-end-->')[0];} catch(e) {console.error('scully could not pare content',e);}</script>
-`;
+  // tslint:disable-next-line:no-unused-expression
```
Comment (SanderElias) [inline scully/renderPlugins/contentRenderPlugin.ts]: It still needs to return the string.

**Comment by Splaktar** [comment]
Doh. Sorry. Will update again soon.

File: scully/renderPlugins/contentRenderPlugin.ts (hunk 2 above)
Comment (Splaktar) [reply scully/renderPlugins/contentRenderPlugin.ts]: Sorry, it was a typo. Fixed.

**Pull request merged**